### PR TITLE
arturia-software-center 2.7.1.2466 (new cask)

### PR DESCRIPTION
**Pull request by sabeelash**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

I unfortunately couldn't get the livecheck working, but other than that, I think everything looks correct. Thank you.

[Arturia](arturia.com) makes hardware and software for music production and this is the official software center used to manage its products.

### CODE REVIEW

File: Casks/a/arturia-software-center.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  zap trash: [
+      "/var/db/receipts/com.Arturia.ArturiaSoftwareCenter.fonts.bom",
+      "/var/db/receipts/com.Arturia.ArturiaSoftwareCenter.fonts.plist",
+      "/var/db/receipts/com.Arturia.ArturiaSoftwareCenter.resources.bom",
+      "/var/db/receipts/com.Arturia.ArturiaSoftwareCenter.resources.plist",
+      "~/Library/Caches/com.Arturia.ArturiaSoftwareCenter",
+      "~/Library/WebKit/com.Arturia.ArturiaSoftwareCenter",
+    "/Library/Arturia/Arturia Software Center",
+  ]
```
Comment (razvanazamfirei) [inline Casks/a/arturia-software-center.rb:27]:
```suggestion
  zap delete: [
        "/Library/Arturia/Arturia Software Center",
        "/Library/ArturiaSC/",
      ],
      trash:  [
        "~/Library/Caches/com.Arturia.ArturiaSoftwareCenter",
        "~/Library/Saved Application State/com.Arturia.ArturiaSoftwareCenter.savedState",
        "~/Library/WebKit/com.Arturia.ArturiaSoftwareCenter",
      ],
      rmdir:  "/Library/Arturia"
```

**Review by razvanazamfirei — CHANGES_REQUESTED** [review]
Thanks for opening this, @sabeelash! I have some suggested changes below!

**Comment by sabeelash** [comment]
Thanks for the suggestions, @razvanazamfirei. I just implemented all of them. First time contributing to any open-source project so thanks for all the help.

**Comment by razvanazamfirei** [comment]
This looks great! 👍 Thanks so much for your first contribution! Know that without contributions like yours, it'd be impossible to keep our project going.

**Review by razvanazamfirei — APPROVED** [review]

**Review by krehel — APPROVED** [review]

**Pull request merged**